cmake/git.cmake

```diff
@@ -0,0 +1,18 @@
+# Use git if it is installed, to set BUILD_VERSION
+
+find_package(Git QUIET)
```

Comment:
I can't see the patch that removes 'find_package(Git' from where this came from.

Reply:
Fixed.